wordclock_plugins/time_in_seconds/plugin.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+import os
+import time
+import time_seconds
+import wordclock_tools.wordclock_colors as wcc
+
+
+class plugin:
+    """
+    Drink mate and do some yoga while watching the time being over and over the same again.
+    'The definition of insanity is doing the same thing over and over and expecting different results.' - Albert Einstein
+    Credits: plotaBot
+    """
+
+    def __init__(self, config):
+        """
+        Initializations for the startup of the current wordclock plugin
+        """
+        # Get plugin name (according to the folder, it is contained in)
+        self.name = os.path.dirname(__file__).split('/')[-1]
+        self.pretty_name = "Seconds"
+        self.description = "Shows the current seconds. The fool thinks no time flies;)"
+
+	self.taw = time_seconds.time_seconds()
+
+	self.bg_color = wcc.BLACK  # default background color
+        self.word_color = wcc.WHITE  # default word color
+        self.minute_color = wcc.BLACK  # default minute color
+
+    def run(self, wcd, wci):
+        """
+        Displays time until aborted by user interaction on pin button_return
+        """
+        # Some initializations of the "previous" second
+        prev_sec = -1
+
+        while True:
+            # Get current time
+            now = datetime.datetime.now()
+            # Check, if a second has passed (to render the new time)
+            if prev_sec < now.second:
+                # Set background color
+                self.show_time(wcd, wci)
+                prev_sec = -1 if now.second == 59 else now.second
```

Comment:
Before reaching "00", the "59" is displayed twice on my clock.... Can you double check this?

Reply:
Nice work, @plotaBot. No worries about the delay. I could track it down and fix it: https://github.com/bk1285/rpi_wordclock/commit/9db8b57c5abadb81ab344eceeb76d419fa05d5eb#diff-d2ecdbd307ac721fde38a199d8de8f62R44